recipes/diffsims/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "diffsims" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 285966371404cb6a200cd11de4cada2784318d1abd05390ff524c95663e52103
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+    - transforms3d
+    - diffpy.structure
+
+test:
+  requires:
+    - pytest
+  imports:
+    - diffsims
+    - diffsims.utils
+    - diffsims.generators
+    - diffsims.signals
+    - diffsims.libraries
+  commands:
+    - pytest --pyargs diffsims
+
+about:
+  home: https://github.com/pyxem/diffsims
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: An open-source Python library for diffraction simulations
+  doc_url: www.pyxem.org
+
+extra:
+  recipe-maintainers:
+    - dnjohnstone
+    - pc494
```

Comment:
@pc494 could you please confirm that you also want to be a maintainer of this package?

Reply:
Yep, I am happy to be a maintainer.